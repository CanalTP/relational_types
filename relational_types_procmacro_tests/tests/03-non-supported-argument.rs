mod test_utils;

use relational_types::*;
use test_utils::*;

#[derive(GetCorresponding)]
pub struct Model {
    #[get_corresponding(nonsupportedargument)]
    animal_to_felines: OneToMany<Animal, Feline>,
}

fn main() {}
