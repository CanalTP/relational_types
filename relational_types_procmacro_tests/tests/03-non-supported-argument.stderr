error: proc-macro derive panicked
 --> $DIR/03-non-supported-argument.rs:6:10
  |
6 | #[derive(GetCorresponding)]
  |          ^^^^^^^^^^^^^^^^
  |
  = help: message: Only `key = "value"` attributes supported.
